spectator-web/src/main/java/com/netflix/spectator/controllers/model/DataPoint.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.controllers.model;
+
+import com.netflix.spectator.api.Measurement;
+
+/**
+ * A serializable Measurement.
```

Comment:
By "serializable" I wasnt refering to java Serialization. I'm open to other ways of doing this. I was following what I'd seen elsewhere. I'd be happy explicitly serializing it rather than converting it into another object only to give to someone else to serialize, but I dont see precedence for doing that (or how one would).